enhancements/console/customize-project-roles.md

```diff
@@ -0,0 +1,176 @@
+---
+title: customize-project-roles
+authors:
+  - "@jerolimov"
+reviewers:
+  - "@spadgett"
+  - "@cvogt"
+  - "@jhadvig"
+approvers:
+  - "@spadgett"
+  - "@cvogt"
+creation-date: 2021-02-25
+last-updated: 2021-03-11
+status: implementable
+---
+
+# Customize Project Access Roles
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [x] Design details are appropriately documented from clear requirements
+- [x] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Motivation
+
+Customers with customized roles want to provide easy access for their project admins to restrict access within a project.
+
+OpenShift Console 3.11 allowed customers to customize the list of roles visible in the membership page. This enhancement will provide equivalent functionality in 4.x.
+
+## Summary
+
+Cluster admins want to give project admins only access to a customized list of predefined roles to force compliance rules or provide more options.
+
+The OpenShift Developer Console provides a simple way to share access to the currently selected project. This change will only affect the project page in the developer perspective. Users can be assigned here to specific roles on the project access tab. Currently only the roles "Admin", "Edit" and "View" are selectable.
+
+The `openshift.io/description` annotation of a `ClusterRole` can be used as label if it is defined.
```

Comment:
I would make it `console.openshift.io/description` if existing roles aren't already using an `openshift.io/description` annotation. There's a desire to not avoid new uses of `openshift.io/description` IIRC.